src/components/MerchantsPage/DescriptionBox.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import styles from './styles.module.scss';
+import merchantIcon from './images/merchantIcon.png';
+
+const Container = styled.div`
```

Comment:
Interesting I have never seen this notation before. Is there a reason why we're doing this as opposed to putting it into an scss?

Reply:
its cleaner and easier to know which css pieces are associated to which! im trying to break out our components into smaller pieces now and i don't to have a css file for each. 

also if there's a css code that is only applicable to that particular component, doesn't make sense to put it all in one shared css file. But I still do have css files in case there's code that can be used for multiple components

Reply:
Wow how interesting. It breaks a lot of very long-standing web idioms of keeping CSS styles separate from function and layout, but I trust your judgment!

Reply:
While I do love styled-components and use it personally and for work, I do believe in keeping our project consistent over cleanliness.  If we are going to move on with using styled-components, then we're going to have to do a slow but big refactor in the current code in styling, otherwise having only a couple of components using this library when it could be done in css. 

That being said, i do prefer the styled-components way of styling and would love to do this refactor with athena's help after we finish responsiveness for the card forms.